rust/agents/relayer/src/settings/mod.rs

```diff
@@ -26,11 +32,13 @@ pub enum GasPaymentEnforcementPolicy {
 #[serde(tag = "type", rename_all = "camelCase")]
 pub struct GasPaymentEnforcementConfig {
     /// The gas payment enforcement policy
+    #[serde(flatten)]
     pub policy: GasPaymentEnforcementPolicy,
-    /// An optional whitelist, where all matching messages will be considered
+    /// An optional matching list, where all matching messages will be considered
     /// as if they have met the gas payment enforcement policy.
```

Comment:
nit: probably best to change this wording so it's clear that a matching message means it'll use this gas payment enforcement policy, and not that they're exempt from a policy (as it used to be)

Reply:
Whoops, fixed.